mobile_insight/analyzer/__init__.py

```diff
@@ -46,3 +46,5 @@
 
 # Visualization
 from log_analyzer import LogAnalyzer
+
+from vr_test_analyzer import VrTestAnalyzer
```

Comment:
We should not put project-specific analyzers `VrTestAnalyzer` into the official MobileInsight release.